tests/integration/cleanup_resources.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+import argparse
+import json
+import os
+import subprocess
+
+
+def cleanup_chaos_mesh(namespace) -> None:
+    print(f"Cleaning up chaos mesh in {namespace}")
+    env = os.environ
+    env["KUBECONFIG"] = os.path.expanduser("~/.kube/config")
+
+    output = subprocess.check_output(
+        f"tests/integration/ha_tests/scripts/destroy_chaos_mesh.sh {namespace}",
+        shell=True,
+        env=env,
+    )
+    for line in output.decode("utf-8").splitlines():
+        print(line)
```

Comment:
Would be better to replace this with a logger

Reply:
Done, thank you